src/nvim/math.c

```diff
@@ -2,6 +2,7 @@
 // it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
 
 #include <math.h>
+
```

Comment:
This looks random. not a big deal tho.

Reply:
"There are no accidents" - Master Oogway

Not random, I've enabled a flag that separates the <>-includes from ""-includes. Can turn it off of course.

Reply:
It's a bug. The filename is math.c and it thinks that it's its own header.